src/debugger/rnDebugSession.ts

```diff
@@ -170,8 +191,10 @@ export class RNDebugSession extends LoggingDebugSession {
             if (logLevel) {
                 logLevel = logLevel.replace(logLevel[0], logLevel[0].toUpperCase());
                 logger.setup(Logger.LogLevel[logLevel], chromeDebugCoreLogs || false);
+                this.cdpProxyLogLevel = LogLevel[logLevel] === LogLevel.Verbose ? LogLevel.Info : LogLevel.None;
             } else {
                 logger.setup(Logger.LogLevel.Log, chromeDebugCoreLogs || false);
+                this.cdpProxyLogLevel = LogHelper.LOG_LEVEL === LogLevel.Trace ? LogLevel.Info : LogLevel.None;
```

Comment:
Does this value exists in `LogHelper`?

Reply:
Yes, it does.`LogHelper . LOG LEVEL` is always available, since it [is fetched](https://github.com/microsoft/vscode-react-native/blob/master/src/extension/log/LogHelper.ts#L28-L31) from the extension's settings.json file or in case this file doesn't exist its default value is "Info".